docs/community.aws.aws_acm_info_module.rst

```diff
@@ -27,9 +27,9 @@ Requirements
 ------------
 The below requirements are needed on the host that executes this module.
 
-- boto
-- boto3
-- python >= 2.6
+- python >= 3.6
+- boto3 >= 1.13.0
+- botocore >= 1.16.0
```

Comment:
Looks like the latest updates from the amazon.aws fragments haven't been applied here.

Reply:
oh hah, I probably had stable-1.5 checked out on my lab while I was working on this since I was also doing the 1.5.1 prep.